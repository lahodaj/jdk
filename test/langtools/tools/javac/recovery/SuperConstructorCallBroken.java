/**
 * @test /nodynamiccopyright/
 * @bug 8259359
 * @summary Check that unexpected super constructor invocation qualifier is attributed
 * @compile/fail/ref=SuperConstructorCallBroken.out -XDshould-stop.at=FLOW -XDdev -XDrawDiagnostics SuperConstructorCallBroken.java
 */
public class SuperConstructorCallBroken extends Undefined1 {
     public SuperConstructorCallBroken(int i) {
         new Undefined2() { public void test(int i) { Undefined3 u; } }.super();
     }
     interface I<T> {}
}
