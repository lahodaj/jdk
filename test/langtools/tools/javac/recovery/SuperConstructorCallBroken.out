SuperConstructorCallBroken.java:7:49: compiler.err.cant.resolve: kindname.class, Undefined1, , 
SuperConstructorCallBroken.java:9:14: compiler.err.cant.resolve.location: kindname.class, Undefined2, , , (compiler.misc.location: kindname.class, SuperConstructorCallBroken, null)
SuperConstructorCallBroken.java:9:55: compiler.err.cant.resolve: kindname.class, Undefined3, , 
3 errors
